modules/exploits/linux/http/eyesofnetwork_autodiscovery_rce.rb

```diff
@@ -164,103 +165,43 @@ def check
     end
   end
 
-def sqli_to_admin_session
+  def sqli_to_admin_session
     sqli_sleep = datastore['SQLI_SLEEP']
```

Comment:
This is only used once, consider just passing in `sqli_opts` similar to `VERBOSE` instead of adding the variable here.